src/components/ShareStructure/ShareStructures.vue

```diff
@@ -66,7 +66,7 @@ export default class ShareStructures extends Mixins(CommonMixin) {
   @Getter getHasRightsOrRestrictions!: boolean
   @Getter isCorrectionFiling!: boolean
   @Getter isAlterationFiling!: boolean
-  @Getter getValidComponentFlags!: ValidComponentsIF
+  @Getter getFlagsCompanyInfo!: FlagsCompanyInfoIF
```

Comment:
The var name updates are definitely useful as someone not that familiar with the code base. 👍